src/Controller/Registration/CancelController.php

```
@@ -16,44 +14,20 @@
-            }
-
-            $this->addFlash('warning', 'Ton inscription a bien été annulée. On espère te voir à un autre évènement !');
-        } catch (\Exception $e) {
-            $this->addFlash('danger', 'Une erreur est survenue lors de l\'annulation de ton inscription. Si le problème persiste, n\'hésite pas à nous contacter.');
-
-            $this->logger->error('An error occurred when canceling registration.', [
-                'registration' => $registration,
-                'payment' => $payment,
-                'exception' => $e,
-            ]);
+            $registrationCancellationResult = $this->registrationCanceller->cancel($registration);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$registrationCancellationResult'. (undefined)

(UnusedLocalVariable)